### PR TITLE
update and separate issue templates

### PR DESCRIPTION
**Pull request by a-tal**
this feature was added by github in late january. should hopefully streamline the issue creation process a bit, compared to the confusing 3-in-1 template currently at least.

I've replaced the default template with a notice to stop and go back to the repo root, and to use the links from there. this will be a slight change in workflow, but I think we can cope 😂 

open to criticism, grammar corrections etc...

~(will have to change the paths to absolute so we can link from gh pages... unless there's some other magic to make a relative path work there)~

~also, if you're looking to try this out to confirm the behaviour, I have issues turned on in my fork for now~

edit: links have been made absolute, testing the creation of new issues won't work anymore

### CODE REVIEW

**Comment by a-tal** [comment]
if you're looking to try this out to confirm the behaviour, I have issues turned on in my fork for now

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Expected
+
+Please provide either the expected return code, correct response body, header value, or some combination thereof. E.g.:
+
+`200`
+
+
+# Checklist
+
```
Comment (ccp-zoetrope) [inline .github/ISSUE_TEMPLATE/bug.md]: There should be some explanation text before the actual checkboxes, E.G.:

"Check all boxes that apply to this issue:"

This comment goes for all "Checklist" headings in the other templates.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- [ ] Bug is documented
```
Comment (ccp-zoetrope) [inline .github/ISSUE_TEMPLATE/bug.md]: This is slightly confusing, maybe wording more like "Bug description is provided"

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- [ ] Request path is documented
```
Comment (ccp-zoetrope) [inline .github/ISSUE_TEMPLATE/bug.md]: I think all check boxes that have the word documented should be changed to the word provided for this bug issue template.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    "bounty": 11000000000,
+    "character_id": 95538921
+  },
+  ...99 other people
+]
+```
+
+# Checklist
+
+- [ ] Feature request is documented
```
Comment (ccp-zoetrope) [inline .github/ISSUE_TEMPLATE/feature_request.md]: "Feature request description is provided"

File: .github/ISSUE_TEMPLATE/inconsistency.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Resolution
+
+Propose a solution to resolve the inconsistency. E.g.:
+
+Change the `id` attribute in the return of `GET /v1/characters/{character_id}/medals/` to `corporation_id`
+
+# Checklist
+
+- [ ] Inconsistency is documented
```
Comment (ccp-zoetrope) [inline .github/ISSUE_TEMPLATE/inconsistency.md]: "Inconsistency description is provided"

File: .github/ISSUE_TEMPLATE/inconsistency.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] Inconsistent route(s) are documented
```
Comment (ccp-zoetrope) [inline .github/ISSUE_TEMPLATE/inconsistency.md]: documented -> provided

File: .github/ISSUE_TEMPLATE/inconsistency.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] Resolution will require a version increase
```
Comment (ccp-zoetrope) [inline .github/ISSUE_TEMPLATE/inconsistency.md]: increase -> bump.

Comment (ccp-zoetrope) [reply .github/ISSUE_TEMPLATE/inconsistency.md]: Also supply a link to [breaking_changes.md](https://github.com/ccpgames/esi-issues/blob/master/breaking_changes.md) as a reference for this and the checkbox below this.

**Comment by a-tal** [comment]
@ccp-zoetrope comments addressed. anyone else before this is merged?

File: README.md [hunk 8]
```diff
@@ -11,6 +11,17 @@ https://esi.tech.ccp.is/
...
+## Opening a new issue
+
+Please use the search function before opening a new issue.
+
+### [Report a new bug](../../issues/new?template=bug.md)
+
+### [Request a new feature](../../issues/new?template=feature_request.md)
+
+### [Report an inconsistency](../../issues/new?template=inconsistency.md)
+
```
Comment (CarbonAlabel) [inline README.md]: Short descriptions should be added next to each of the links, so people can know what is considered a bug/feature request/inconsistency. Also, what should be done if their issue doesn't fit into any of the categories?

File: README.md [hunk 9]
```diff
@@ -11,6 +11,17 @@ https://esi.tech.ccp.is/
 **Please do not open issues for anything security related here, send an email to security@ccpgames.com instead.**
 
 
+## Opening a new issue
+
+Please use the search function before opening a new issue.
```
Comment (CarbonAlabel) [inline README.md]: Maybe specifically remind people to search through closed issues as well?

File: .github/ISSUE_TEMPLATE/inconsistency.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+Change the `id` attribute in the return of `GET /v1/characters/{character_id}/medals/` to `corporation_id`
+
+# Checklist
+
+Check all boxes that apply to this issue:
+
+- [ ] Inconsistency description is provided
+- [ ] Inconsistent route(s) are provided
+- [ ] Resolution [will require a version bump](https://github.com/ccpgames/esi-issues/blob/master/breaking_changes.md)
+- [ ] Resolution [does not require a version bump](https://github.com/ccpgames/esi-issues/blob/master/breaking_changes.md)
```
Comment (CarbonAlabel) [inline .github/ISSUE_TEMPLATE/inconsistency.md]: Could these be changed to relative links?

File: README.md (hunk 8 above)
Comment (a-tal) [reply README.md]: added short descriptions, thanks

File: README.md (hunk 9 above)
Comment (a-tal) [reply README.md]: good call, added

File: .github/ISSUE_TEMPLATE/inconsistency.md (hunk 10 above)
Comment (a-tal) [reply .github/ISSUE_TEMPLATE/inconsistency.md]: updated

**Comment by a-tal** [comment]
all comments addressed at this point, will wait for sign offs and if anyone else has something to add

**Pull request merged**